setup.py

```diff
@@ -24,6 +24,15 @@
     long_description_content_type="text/markdown",
     url="https://github.com/SymbiFlow/prjxray",
     packages=['prjxray'],
+    install_requires=[
+        'fasm',
+        'intervaltree',
+        'numpy',
+        # FIXME: isolate lib.py and node_lookup.py from the package to remove this dependency
```

Comment:
It's specifically `lib.NodeLookup`, not all of `lib.py` that is a problem.  Also please fill an issue to track fixing this, and link it here.